src/_pytest/config/__init__.py

```diff
@@ -1260,6 +1260,7 @@ def create_terminal_writer(config: Config, *args, **kwargs) -> TerminalWriter:
     tw = TerminalWriter(*args, **kwargs)
     if config.option.color == "yes":
         tw.hasmarkup = True
+    tw.code_highlight = False if config.option.code_highlight == "yes" else True
```

Comment:
```suggestion
    tw.code_highlight = config.option.code_highlight == "yes"
```

(I have a [wiki page](https://github.com/asottile/scratch/wiki/PythonHintsForNoobs#useless-branches) about this)

Reply:
Fixed this.

Reply:
looks like this didn't get fixed?

Reply:
Fixed.